iml-gui/crate/src/components/command_modal.rs

```diff
@@ -588,6 +660,42 @@ fn perform_click(select: &mut Select, id: TypedId) -> bool {
     }
 }
 
+fn to_load_cmd(model: &Model, cmd_id: u32) -> bool {
+    // load it if the command is not found or is not complete
+    if let Some(cmd) = model.commands.get(&cmd_id) {
+        !cmd.complete
+    } else {
+        true
+    }
```

Comment:
```suggestion
    model.commands.get(&cmd_id).map(|c| !c.complete).unwrap_or(true)
```

Reply:
It was used to be _exactly this_ expression, however I find it more cumbersome than the explicit if.